### PR TITLE
fix: update links from goerli to holesky

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Link to EthStaker Goerli launchpad no longer functional; updates to use Holesky link.
- Updates English string only; remainder to be patched via Crowdin flow.

## Related Issue
None filed

### CODE REVIEW

File: public/content/developers/docs/networks/index.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Goerli is a testnet for testing validating and staking. The Goerli network is op
 - [Coinbase Wallet Faucet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli faucet](https://faucet.chainstack.com/goerli-faucet)
 
-To launch a Validator on Goerli testnet, use ethstaker's ["cheap goerli validator" launchpad](https://goerli.launchpad.ethstaker.cc/en/).
+To launch a Validator on Holesky testnet, use ethstaker's ["cheap goerli validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/).
```
Comment (corwintines) [inline public/content/developers/docs/networks/index.md:95]:
```suggestion
To launch a Validator on Holesky testnet, use ethstaker's ["cheap Holesky validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/).
```

Comment (corwintines) [inline public/content/developers/docs/networks/index.md:95]:
```suggestion
To launch a Validator on Holesky testnet, use ethstaker's ["cheap Holesky validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/).
```

File: public/content/translations/cs/developers/docs/networks/index.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Goerli je testovací síť pro testování validace a uzamčení. Síť Goerli j
 - [Coinbase Wallet Faucet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli faucet](https://faucet.chainstack.com/goerli-faucet)
 
-Chcete-li spustit validátor na testovací síti Goerli, použijte [launchpad „cheap goerli validator“](https://goerli.launchpad.ethstaker.cc/en/) od ethstaker.
+Chcete-li spustit validátor na testovací síti Goerli, použijte [launchpad „cheap goerli validator“](https://holesky.launchpad.ethstaker.cc/en/) od ethstaker.
```
Comment (corwintines) [inline public/content/translations/cs/developers/docs/networks/index.md:94]:
```suggestion
Chcete-li spustit validátor na testovací síti Holesky, použijte [launchpad „cheap Holesky validator“](https://holesky.launchpad.ethstaker.cc/en/) od ethstaker.
```

File: public/content/translations/de/developers/docs/networks/index.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ Goerli ist ein Testnet zum Testen, Validieren und Staking. Das Goerli-Netzwerk i
 - [Coinbase Wallet Faucet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli-Faucet](https://faucet.chainstack.com/goerli-faucet)
 
-Um einen Validator im Goerli-Testnet zu starten, verwenden Sie das [Launchpad "cheap goerli validator"](https://goerli.launchpad.ethstaker.cc/en/) von ethstaker.
+Um einen Validator im Goerli-Testnet zu starten, verwenden Sie das [Launchpad "cheap goerli validator"](https://holesky.launchpad.ethstaker.cc/en/) von ethstaker.
```
Comment (corwintines) [inline public/content/translations/de/developers/docs/networks/index.md:91]:
```suggestion
Um einen Validator im Holesky-Testnet zu starten, verwenden Sie das [Launchpad "cheap Holesky validator"](https://holesky.launchpad.ethstaker.cc/en/) von ethstaker.
```

File: public/content/translations/es/developers/docs/networks/index.md [hunk 4]
```diff
@@ -90,7 +90,7 @@ Goerli es una red de prueba para probar la validación y la participación. La r
 - [Faucet Chainstack Goerli](https://faucet.chainstack.com/goerli-faucet)
 - [Blockscout](https://eth-goerli.blockscout.com/)
 
-Para iniciar un validador en la red de prueba de Goerli, utilice la plataforma de lanzamiento de [«cheap goerli validator»](https://goerli.launchpad.ethstaker.cc/en/) de ethstaker.
+Para iniciar un validador en la red de prueba de Goerli, utilice la plataforma de lanzamiento de [«cheap goerli validator»](https://holesky.launchpad.ethstaker.cc/en/) de ethstaker.
```
Comment (corwintines) [inline public/content/translations/es/developers/docs/networks/index.md:93]:
```suggestion
Para iniciar un validador en la red de prueba de Holesky, utilice la plataforma de lanzamiento de [«cheap Holesky validator»](https://holesky.launchpad.ethstaker.cc/en/) de ethstaker.
```

File: public/content/translations/fa/developers/docs/networks/index.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ Goerli یک شبکه‌ تست برای آزمایش اعتبارسنجی و س
 - [فاست کیف پول Coinbase | Sepolia](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [فاست Chainstack Sepolia](https://faucet.chainstack.com/goerli-faucet)
 
-برای راه‌اندازی اعتبارسنج بر روی شبکه تست گورلی (Goerli)، از [سکوی پرتاپ"اعتبار سنج ارزان گورلی"](https://goerli.launchpad.ethstaker.cc/en/) که توسط جامعه Ethstaker ارائه میشود استفاده کنید.
+برای راه‌اندازی اعتبارسنج بر روی شبکه تست گورلی (Goerli)، از [سکوی پرتاپ"اعتبار سنج ارزان گورلی"](https://holesky.launchpad.ethstaker.cc/en/) که توسط جامعه Ethstaker ارائه میشود استفاده کنید.
```
Comment (corwintines) [inline public/content/translations/fa/developers/docs/networks/index.md:91]:
```suggestion
برای راه‌اندازی اعتبارسنج بر روی شبکه تست گورلی (Holesky)، از [سکوی پرتاپ"اعتبار سنج ارزان گورلی"](https://holesky.launchpad.ethstaker.cc/en/) که توسط جامعه Ethstaker ارائه میشود استفاده کنید.
```

File: public/content/translations/fr/developers/docs/networks/index.md [hunk 6]
```diff
@@ -91,7 +91,7 @@ Goerli est un réseau de test qui permet de tester, de valider et de mettre en j
 - [Coinbase Wallet Faucet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Robinet Goerli Chainstack](https://faucet.chainstack.com/goerli-faucet)
 
-Pour lancer un validateur sur le réseau de test Goerli, utilisez la [plateforme de lancement "validateur goerli bon marché"](https://goerli.launchpad.ethstaker.cc/en/) d'ethstaker.
+Pour lancer un validateur sur le réseau de test Goerli, utilisez la [plateforme de lancement "validateur goerli bon marché"](https://holesky.launchpad.ethstaker.cc/en/) d'ethstaker.
```
Comment (corwintines) [inline public/content/translations/fr/developers/docs/networks/index.md:94]:
```suggestion
Pour lancer un validateur sur le réseau de test Holesky, utilisez la [plateforme de lancement "validateur Holesky bon marché"](https://holesky.launchpad.ethstaker.cc/en/) d'ethstaker.
```

File: public/content/translations/hu/developers/docs/networks/index.md [hunk 7]
```diff
@@ -91,7 +91,7 @@ A Goerli egy olyan teszthálózat, ahol a validálást és a letétbe helyezést
 - [Coinbase Wallet csap | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli csap](https://faucet.chainstack.com/goerli-faucet)
 
-Ha szeretne egy validátort indítani a Goerli teszthálózaton, akkor használja az ethstaker [olcsó goerli validátor launchpad-et](https://goerli.launchpad.ethstaker.cc/en/).
+Ha szeretne egy validátort indítani a Goerli teszthálózaton, akkor használja az ethstaker [olcsó goerli validátor launchpad-et](https://holesky.launchpad.ethstaker.cc/en/).
```
Comment (corwintines) [inline public/content/translations/hu/developers/docs/networks/index.md:94]:
```suggestion
Ha szeretne egy validátort indítani a Holesky teszthálózaton, akkor használja az ethstaker [olcsó Holesky validátor launchpad-et](https://holesky.launchpad.ethstaker.cc/en/).
```

File: public/content/translations/it/developers/docs/networks/index.md [hunk 8]
```diff
@@ -90,7 +90,7 @@ Goerli è una rete di prova per testare la convalida e lo staking. La rete Goerl
 - [Faucet Coinbase Wallet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Faucet Gorli di Chainstack](https://faucet.chainstack.com/goerli-faucet)
 
-Per lanciare un Validatore sulla rete di prova Goerli, usa il ["cheap goerli validator" launchpad](https://goerli.launchpad.ethstaker.cc/en/) di ethstaker.
+Per lanciare un Validatore sulla rete di prova Goerli, usa il ["cheap goerli validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/) di ethstaker.
```
Comment (corwintines) [inline public/content/translations/it/developers/docs/networks/index.md:93]:
```suggestion
Per lanciare un Validatore sulla rete di prova Holesky, usa il ["cheap Holesky validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/) di ethstaker.
```

File: public/content/translations/ja/developers/docs/networks/index.md [hunk 9]
```diff
@@ -90,7 +90,7 @@ Goerliは、バリデーションやステーキングのテストを行うた
 - [Coinbase Wallet Faucet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli faucet](https://faucet.chainstack.com/goerli-faucet)
 
-Goerliテストネットでバリデータを起動するには、ethstakerの["cheap goerli validator" launchpad](https://goerli.launchpad.ethstaker.cc/en/)を使用してください。
+Goerliテストネットでバリデータを起動するには、ethstakerの["cheap goerli validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/)を使用してください。
```
Comment (corwintines) [inline public/content/translations/ja/developers/docs/networks/index.md:93]:
```suggestion
Holeskyテストネットでバリデータを起動するには、ethstakerの["cheap Holesky validator" launchpad](https://holesky.launchpad.ethstaker.cc/en/)を使用してください。
```

File: public/content/translations/pt-br/developers/docs/networks/index.md [hunk 10]
```diff
@@ -88,7 +88,7 @@ Goerli é a rede de testes usada para testar a validação e staking. A rede Goe
 - [Coinbase Wallet Faucet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Faucet Chainstack Goerli](https://faucet.chainstack.com/goerli-faucet)
 
-Para iniciar um Validador na rede de testes Goerli, use a barra de inicialização ["validador goerli barato"](https://goerli.launchpad.ethstaker.cc/en/) do ethstaker.
+Para iniciar um Validador na rede de testes Goerli, use a barra de inicialização ["validador goerli barato"](https://holesky.launchpad.ethstaker.cc/en/) do ethstaker.
```
Comment (corwintines) [inline public/content/translations/pt-br/developers/docs/networks/index.md:91]:
```suggestion
Para iniciar um Validador na rede de testes Holesky, use a barra de inicialização ["validador Holesky barato"](https://holesky.launchpad.ethstaker.cc/en/) do ethstaker.
```

File: public/content/translations/ru/developers/docs/networks/index.md [hunk 11]
```diff
@@ -91,7 +91,7 @@ Goerli — тестовая сеть для проверки валидации
 - [Кран Coinbase Wallet | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Кран Goerli от Chainstack](https://faucet.chainstack.com/goerli-faucet)
 
-Чтобы стать валидатором в тестовой сети Goerli, используйте [панель запуска cheap goerli validator](https://goerli.launchpad.ethstaker.cc/en/) от EthStaker.
+Чтобы стать валидатором в тестовой сети Goerli, используйте [панель запуска cheap goerli validator](https://holesky.launchpad.ethstaker.cc/en/) от EthStaker.
```
Comment (corwintines) [inline public/content/translations/ru/developers/docs/networks/index.md:94]:
```suggestion
Чтобы стать валидатором в тестовой сети Holesky, используйте [панель запуска cheap Holesky validator](https://holesky.launchpad.ethstaker.cc/en/) от EthStaker.
```

File: public/content/translations/tr/developers/docs/networks/index.md [hunk 12]
```diff
@@ -88,7 +88,7 @@ Goerli doğrulama ve hisselemeyi test etme amaçlı bir test ağıdır. Goerli a
 - [Coinbase Cüzdanı Musluğu | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli musluğu](https://faucet.chainstack.com/goerli-faucet)
 
-Goerli test ağında bir Doğrulayıcı başlatmak için, ethstaker'ın ["ucuz goerli doğrulayıcısı" başlama noktasını](https://goerli.launchpad.ethstaker.cc/en/) kullanın.
+Goerli test ağında bir Doğrulayıcı başlatmak için, ethstaker'ın ["ucuz goerli doğrulayıcısı" başlama noktasını](https://holesky.launchpad.ethstaker.cc/en/) kullanın.
```
Comment (corwintines) [inline public/content/translations/tr/developers/docs/networks/index.md:91]:
```suggestion
Holesky test ağında bir Doğrulayıcı başlatmak için, ethstaker'ın ["ucuz Holesky doğrulayıcısı" başlama noktasını](https://holesky.launchpad.ethstaker.cc/en/) kullanın.
```

File: public/content/translations/zh-tw/developers/docs/networks/index.md [hunk 13]
```diff
@@ -90,7 +90,7 @@ Goerli 是測試驗證和質押的測試網。 Goerli 測試網對想要運行
 - [Coinbase 錢包水龍頭 | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli 水龍頭](https://faucet.chainstack.com/goerli-faucet)
 
-要在 Goerli 測試網上啟動驗證者，請使用 ethstaker 的[「cheap goerli validator（實惠的 goerli 驗證者）」啟動面板](https://goerli.launchpad.ethstaker.cc/en/)。
+要在 Goerli 測試網上啟動驗證者，請使用 ethstaker 的[「cheap goerli validator（實惠的 goerli 驗證者）」啟動面板](https://holesky.launchpad.ethstaker.cc/en/)。
```
Comment (corwintines) [inline public/content/translations/zh-tw/developers/docs/networks/index.md:93]:
```suggestion
要在 Holesky 測試網上啟動驗證者，請使用 ethstaker 的[「cheap Holesky validator（實惠的 goerli 驗證者）」啟動面板](https://holesky.launchpad.ethstaker.cc/en/)。
```

File: public/content/translations/zh/developers/docs/networks/index.md [hunk 14]
```diff
@@ -90,7 +90,7 @@ Goerli 是用于测试验证和质押的测试网。 Goerli 网络对想要运
 - [Coinbase Wallet 水龙头 | Goerli](https://coinbase.com/faucets/ethereum-goerli-faucet)
 - [Chainstack Goerli 水龙头](https://faucet.chainstack.com/goerli-faucet)
 
-要在 Goerli 测试网上启动验证者，请使用 ethstaker 的 ["cheap goerli validator" 启动板](https://goerli.launchpad.ethstaker.cc/en/)。
+要在 Goerli 测试网上启动验证者，请使用 ethstaker 的 ["cheap goerli validator" 启动板](https://holesky.launchpad.ethstaker.cc/en/)。
```
Comment (corwintines) [inline public/content/translations/zh/developers/docs/networks/index.md:93]:
```suggestion
要在 Holesky 测试网上启动验证者，请使用 ethstaker 的 ["cheap Holesky validator" 启动板](https://holesky.launchpad.ethstaker.cc/en/)。
```

**Review by corwintines — COMMENTED** [review]
@wackerow should these also go from Goerli to Holesky? Or maybe this sentence should just be removed at this point?

**Comment by wackerow** [comment]
Good catch, thanks @corwintines! Committed that string update for the English version.. hoping to let the Crowdin process handle the non-English versions since this is lower priority. Should be good now

**Review by corwintines — APPROVED** [review]
LGTM

**Pull request merged**